tendrl/commons/flows/expand_cluster/utils.py

```diff
@@ -0,0 +1,58 @@
+import copy
```

Comment:
https://github.com/Tendrl/commons/blob/develop/tendrl/commons/flows/create_cluster/utils.py#L125 to https://github.com/Tendrl/commons/blob/develop/tendrl/commons/flows/create_cluster/utils.py#L138 not needed in this case